src/JuMP.jl

```diff
@@ -32,8 +32,8 @@ export
     affToStr, quadToStr, conToStr, chgConstrRHS,
 
 # Macros and support functions
-    @addConstraint, @addConstraints, @defVar,
-    @defConstrRef, @setObjective, addToExpression, @defExpr,
+    @addConstraint, @defConstraint, @addConstraints, @defConstraints,
```

Comment:
I'm not really convinced that we should provide a nice syntax for this. Someone who doesn't understand JuMP's internals could easily read ``@defConstraint`` as defining a constraint that will be enforced by the model (i.e., added to the model). The distinction between ``@defConstraint`` and ``@addConstraint`` is a whole new unintuitive concept that we would need to add to the manual.